pkg/api/universal/conversation_middleware.go

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright 2025 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package universal
+
+import (
+	"fmt"
+	"regexp"
+
+	piiscrubber "github.com/aavaz-ai/pii-scrubber"
+)
+
+// StreamingPIIScrubber handles real-time PII scrubbing for streaming content
+type StreamingPIIScrubber struct {
+	buffer     []byte
+	windowSize int // Look-ahead window for PII detection
+	scrubber   piiscrubber.Scrubber
+}
+
+// PIIPattern represents a PII detection pattern
+type PIIPattern struct {
```

Comment:
Used?

Reply:
not really, thought I would but no, I'll remove